install-dev/theme/views/process.php

```diff
@@ -113,9 +113,6 @@
 		<button type="button" class="btn-facebook" onclick="psinstall_facebook_click();">
 			<i></i> <?php echo $this->translator->trans('Share', array(), 'Install'); ?>
 		</button>
-		<button type="button" class="btn-google-plus" onclick="psinstall_google_click();">
```

Comment:
Need to remove the function `psinstall_google_click` from JS !

Reply:
Sorry @Progi1984  I am getting very old ... Thanks !